modelling_opponent/MouseMonitor.py

```diff
@@ -6,7 +6,7 @@ def __init__(self, video, mouse_id):
         self.mouse_id = mouse_id
         self.video_analyser = video
 
-    def getDecision(self, zones_list):
+    def getDecision(self, zones_list,state):
```

Comment:
Not sure about this one. The State argument is not needed (As far as I remember, we discussed it and we agreed). Why are you adding it back? Since this is the only change in the MouseMonitor, I suggest to stay with the version on main.

Reply:
i got an error saying the number of arguments to the mouse monitor from is not consistent.  

 File "C:\Users\EngelHardBlab.MEDICINE\Downloads\PrisonerDilemmaPy_(4)\PrisonerDilemmaPy\Experiment_Launcher_code\ExperimentManager.py", line 253, in start_streaming_exp
    mouselocation = mouse1.getDecision(zone_activations, currentstate)
TypeError: MouseMonitor.getDecision() takes 2 positional arguments but 3 were given